### ISSUE TITLE
recording_rules page is confusing

### ISSUE THREAD

**Issue by jpds** [prometheus/docs#1212]
The page at: https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/ is highly confusing.

1) It shows an example rules file
2) Immediately goes to `<rule_group>` where there should be an accompanying explanation (also loses the indentation).
3) Shows a `<rule>` example which seems to suggest that `record` is a required field (not in `[]`)
4) Then an alert example is shown where (the required) `record` is gone and replaced with `alert`

**Comment by andrechalella** [comment]
Yes. Also lots of info that pertain to "Alerting rules" are in the "Recording rules" page. IMO, since both pages are relatively small and have so much in common, they should be merged.

**Comment by jan--f** [comment]
Hello from the bug scrub!

I think this has been much improved since this issue was opened, so I'll close this for now. Please reopen is there is any more work to do.
